portfolio/src/main/java/com/google/sps/servlets/QueryHelper.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+/** 
+  This class handles query operations to the datastore.
+*/
+public class QueryHelper {
+
+  private static final String TIMESTAMP = "timestamp";
+  private Query myQuery;
+  private DatastoreService myDatastore;
+  private PreparedQuery myResults;
+  private String queryString;
+
+  public QueryHelper(String queryString) {
```

Comment:
Yes I think so. Initially I made it like that keeping flexibility in mind.. But since the only thing I'm using the class for in this project is the Comment function, is it ok to make the class just work for Comment queries?

Reply:
If I made the class more specific to the comment function, I could then move more query/database functionality into it.